core/src/main/java/com/novoda/noplayer/model/VideoPosition.java

```diff
@@ -1,118 +0,0 @@
-package com.novoda.noplayer.model;
-
-import java.io.Serializable;
-import java.util.concurrent.TimeUnit;
-
-public final class VideoPosition implements Comparable<VideoPosition>, Serializable {
-
-    public static final VideoPosition BEGINNING = new VideoPosition(0);
-    public static final VideoPosition INVALID = new VideoPosition(-1);
-
-    private static final long SECOND_IN_MILLIS = TimeUnit.SECONDS.toMillis(1);
-    private final long positionInMillis;
-
-    public static VideoPosition fromMillis(long positionInMillis) {
-        return new VideoPosition(positionInMillis);
-    }
-
-    public static VideoPosition fromSeconds(double positionInSeconds) {
-        return new VideoPosition((long) (positionInSeconds * SECOND_IN_MILLIS));
-    }
-
-    private VideoPosition(long positionInMillis) {
-        this.positionInMillis = positionInMillis;
-    }
-
-    public boolean isAfter(VideoPosition otherPosition) {
-        return positionInMillis > otherPosition.positionInMillis;
-    }
-
-    public boolean isBefore(VideoPosition otherPosition) {
-        return positionInMillis < otherPosition.positionInMillis;
-    }
-
-    public long inMillis() {
-        return positionInMillis;
-    }
-
-    public int inImpreciseMillis() {
-        return (int) positionInMillis;
-    }
-
-    public int inImpreciseSeconds() {
```

Comment:
Wondering how much all the methods on this type were used by user(s) of the library? If they're particularly useful we might consider keeping them available as static. This is the one time I'd be willing to suggest a big bundle of static functions as a possibly good thing 😆

Reply:
When I do a local release I'll check to see how much we use these and possibly add some sort of utility class to help out. Although @ataulm did mention that some of these are confusing / incorrect https://github.com/novoda/no-player/issues/101

Reply:
I think #101 could become obsolete after some of these changes are complete.